File: 1/1.c
//O(n^2)
int* twoSum(int* nums, int numsSize, int target) {
    int i = 0, j = 0;
    int* res;
    res = (int*)malloc(2 * sizeof(int));
    for(i=0; i<numsSize-1; i++){
        for(j=i+1; j<numsSize; j++){
            if(nums[i]+nums[j]==target){
                res[0] = i;
                res[1] = j;
                return res; 
                //为啥这句不写就不对啊。。
            }
        }
    }
    return res;
}

File: 1/1.py
# time complexity: O(n)
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        di = {}
        for i, no in enumerate(nums):
            res_num = target - no
            if res_num in di:
                return [i, di[res_num]]
            else:
                di[no] = i
